tslint.json

```diff
@@ -30,11 +30,13 @@
     "no-string-based-set-timeout": true,
     "react-no-dangerous-html": [
       true,
-      {
-        "file": "./src/app/containers/Html.tsx",
-        "method": "render",
-        "comment": "string comes from trusted source - our initial state"
-      }
+      [
```

Comment:
wow, the fix was easy 😄 I didn't look internally

Reply:
Hey I took this [fix](https://github.com/microsoft/tslint-microsoft-contrib/issues/880#issue-463671661) from you. Come on Nishchal.

Reply:
wow 😄 I forgot about it.